### ISSUE TITLE
Question : What was your process during the recording ?

### ISSUE THREAD

**Issue by R-Raph** [UoA-eResearch/leapmotion_gestures#20]
I understood how the record.py and many_to_one files works but I would like to know if there is a way to execute the gesture during the recording that optimizes the learning performance.

for example in the case of a stationary gesture: as the learning takes into account the movements in the gesture, is it necessary to remain totally motionless during a certain time and to multiply the recordings of the same gesture under different angles/rotations?

Is it necessary to start the recording directly in the required gesture or will the beginning and end parasite sections will be filtered by the learning?

will the learning be of better quality if we show the creation of the gesture (no gesture to thumb up so that the stationary thumb up is better recognized) ? 

Is there anything else important in the process of recording that i should know ?

Thank you in advance for the answers

**Comment by neon-ninja** [comment]
Hi!

>stationary gesture

What you might think is a stationary gesture, isn't perfectly stationary (one frame) for the model. No-one's hands are perfectly steady (even if you're a surgeon!), and there's also noise from the imperfect leap motion processing. So, even a stationary gesture involves some changes in the variables used. Which is useful! The model can then learn to ignore slight jittering / noise

>will the beginning and end parasite sections will be filtered by the learning

Don't worry too much about this. The bulk of the training data will be fine. A training dataset composed of 99% well labelled stuff and 1% junk still results in a model with good accuracy.

>will the learning be of better quality if we show the creation of the gesture (no gesture to thumb up so that the stationary thumb up is better recognized) ?

It shouldn't matter

>Is there anything else important in the process of recording that i should know ?

Get multiple people to provide recordings! Everyone has different hands, and slightly different ways of doing gestures, so it's a good way to avoid overfit. The original version of the model, trained only on @aleathwick's hands, only worked for his hands.

**Comment by R-Raph** [comment]
alright thanks for the answers.
